Fix:

warning: a function declaration without a prototype is deprecated in all versions of C [-Wstrict-prototypes]
--- osx.c.orig	2005-07-28 12:29:07.000000000 -0500
+++ osx.c	2024-10-02 01:39:23.000000000 -0500
@@ -38,7 +38,7 @@
  * \return Returns an allocated pointer to the network location name. The
  * programmer must free the pointer.
  */
-char *osx_location()
+char *osx_location(void)
 {
 	FILE *scselect;
 	char buf[BUFSIZ], location[BUFSIZ];
--- osx.h.orig	2005-07-28 12:29:07.000000000 -0500
+++ osx.h	2024-10-02 01:39:31.000000000 -0500
@@ -26,7 +26,7 @@
 #include <stdarg.h>
 #endif
 
-char *osx_location();
+char *osx_location(void);
 
 #ifdef HAVE_GROWLNOTIFY
 int osx_notify(char*,char*,...);
--- smtp.c.orig	2005-07-28 12:29:07.000000000 -0500
+++ smtp.c	2024-10-02 01:40:21.000000000 -0500
@@ -64,7 +64,7 @@
  *
  * \returns A pointer to the string with the last message from the server
  */
-char *smtp_last_message()
+char *smtp_last_message(void)
 {
 	return smtp_stored_message;
 }
